changes/changes-8.8.0.md

```diff
@@ -75,7 +75,7 @@ Release Notes.
   low down metrics flush period added in 8.7.0.
 * Fix `Slow SQL sampling` not accurate, due to TopN works conflict with cache read-then-clear mechanism.
 * The persistent cache is only read when necessary.
-* Add component definition for `Alibaba Fastjson`.
+* Add component definition for `Alibaba Fastjson` and `Jackson`.
```

Comment:
You should not move to here. The original version is correct.

Reply:
This is main repository's release log, which has started. https://lists.apache.org/thread.html/rde5ccf89acdf60215b4409097273bf171b81fdda8d22370f4a33414a%40%3Cdev.skywalking.apache.org%3E

Reply:
All change logs in `changes` folder are archived versions, unless there are typo, they should not be changed.

Reply:
OK, will reset it , but the java-agent verison is 8.8.0-SNAPSHOT, is OK?

Reply:
Java agent is going to use its own version from this month. There are no relationship, version relationship is described in https://skywalking.apache.org/docs/main/latest/en/setup/service-agent/agent-compatibility/

Reply:
Read this, if you want to see what is happening. https://github.com/apache/skywalking/discussions/7548